src/asm/rgbasm.5

```diff
@@ -394,13 +394,13 @@ Most of them return a string, however some of these functions actually return an
 .It Fn STRCMP str1 str2 Ta Returns -1 if Ar str1 No is alphabetically lower than Ar str2 No , zero if they match, 1 if Ar str1 No is greater than Ar str2 .
 .It Fn STRIN str1 str2 Ta Returns the first position of Ar str2 No in Ar str1 No or zero if it's not present Pq first character is position 1 .
 .It Fn STRRIN str1 str2 Ta Returns the last position of Ar str2 No in Ar str1 No or zero if it's not present Pq first character is position 1 .
-.It Fn STRSUB str pos len Ta Returns a substring from Ar str No starting at Ar pos No (first character is position 1) and Ar len No characters long.
+.It Fn STRSUB str pos len Ta Returns a substring from Ar str No starting at Ar pos No (first character is position 1) and Ar len No characters long. Zero or negative Ar pos No counts from the end, as if Qo STRLEN(str) Qc were added to it.
```

Comment:
*Why did they use 1-based indexes, oh why why why...*

I don't like that 0 maps to the end of the string; I think it would make more sense for it to be -1. This is furthered by `STRR?IN`  returning 0 when failing to find the character in question.
What do you think?

Reply:
I'm not sure. 0 for the end makes `pos` consistently act like the current `STRLEN(str) - pos` solution. But -1 being the last character when 1 is the first also sounds intuitive. @aaaaaa123456789 What do you think?

Reply:
Actually I agree with you, counting positions forwards from 1 and backwards from -1 makes sense. That's been updated.